content/en/docs/tasks/administer-cluster/sysctl-cluster.md

```diff
@@ -53,6 +53,14 @@ same node. This means that setting a _safe_ sysctl for one pod
 - must not allow to gain CPU or memory resources outside of the resource limits
   of a pod.
 
+{{< note >}}
+Starting from Kubernetes version 1.23, kubelet supports the use of "/" or "." as separators for sysctl.
+Sysctls can be converted to a slash as a separator.
+For example, the sysctl `kernel.shm_rmid_forced` can also be written as `kernel/shm_rmid_forced` with a slash as a separator.
+For more sysctl parameter conversion method details, please refer to
+[Linux sysctl documentation](https://man7.org/linux/man-pages/man5/sysctl.d.5.html)
```

Comment:
```suggestion
the page [sysctl.d(5)](https://man7.org/linux/man-pages/man5/sysctl.d.5.html) from
the Linux man-pages project.
```

I'd consider linking to https://www.freedesktop.org/software/systemd/man/sysctl.d.html in the systemd docs instead. (I'm not sure which Linux package owns this convention).

Reply:
This looks more accurate, thanks.